src/components/dialogs/ag-grid-table-rhf/cell-editors/number-editors.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { forwardRef, useImperativeHandle, useState } from 'react';
+import { Input } from '@mui/material';
+
+const NumberEditor = forwardRef(({ ...props }, ref) => {
+    const [value, setValue] = useState(props.value ?? null);
+
+    const handleChange = (event) => {
+        setValue(event.target.value);
+    };
+
+    useImperativeHandle(
+        ref,
+        () => {
+            return {
+                getValue: () => {
+                    return value;
+                },
+            };
+        },
+        [value]
+    );
```

Comment:
what is this ?

Reply:
Why this time it's getValue returns value without 's' ?